### DISCUSSION TITLE
table column dimenssions

### DISCUSSION THREAD

**Discussion by moathOSA** [apache/superset#19516] [Q&A / Help]
i have an issue in table column dimensions , i can't specify the column width and height to view only small data and view all data only on click !!

also table columns not supporting JSON format like in SqlLab

![table_column_dimensions_issue](https://user-images.githubusercontent.com/60176683/161419862-32487ebd-e897-4d7e-9169-06e65b3d06ed.jpg)

superset version 1.4.1
### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by srinify** [comment]
Hey there @moathOSA these are both good suggestions but I will say that they aren't bugs per say (the functionality is working as originally designed). Moving to discussions to see if others want to weigh on the discussion around improvements!

**Reply by kasiazjc** [reply]
I agree that tables are not working too well currently, but Srini is right that it is the implemented behavior. 

We are looking to switch to ant-d tables which would provide more flexibility, but cannot say when it will happen as it is quite a big effort with the audit etc.
